<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>booking_dialog_add</class>
 <widget class="QDialog" name="booking_dialog_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>booking_dialog_add</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52,54)</string>
  </property>
  <widget class="QLineEdit" name="UserLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: rgb(211, 215, 207);
	border: 2px solid rgb(211, 215, 207);
	border-radius: 20px;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(63, 67, 68);
	font: 13pt &quot;Ubuntu&quot;;
}
QLineEdit::hover{
	border: 2px solid rgb(85, 87, 83)
}
QLineEdit::focus{
	border: 2px solid rgb(12, 124, 164)
}</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserCountLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: rgb(211, 215, 207);
	border: 2px solid rgb(211, 215, 207);
	border-radius: 20px;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(63, 67, 68);
	font: 13pt &quot;Ubuntu&quot;;
}
QLineEdit::hover{
	border: 2px solid rgb(85, 87, 83)
}
QLineEdit::focus{
	border: 2px solid rgb(12, 124, 164)
}</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="DateLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: rgb(211, 215, 207);
	border: 2px solid rgb(211, 215, 207);
	border-radius: 20px;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(63, 67, 68);
	font: 13pt &quot;Ubuntu&quot;;
}
QLineEdit::hover{
	border: 2px solid rgb(85, 87, 83)
}
QLineEdit::focus{
	border: 2px solid rgb(12, 124, 164)
}</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="date_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 215, 207);
font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_count_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 215, 207);
font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>User count</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 215, 207);
font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignInClicked">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(211, 215, 207);
	border: 2px solid rgb(211, 215, 207);
	border-radius:20px;
	background-color: rgb(63, 67, 68);
	font: 13pt &quot;Ubuntu&quot;;
}
QPushButton::hover{
	border: 2px solid rgb(85, 87, 83);
	background-color: rgb(45, 47, 48)
}
</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="Day">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	color: rgb(211, 215, 207);
	border: 2px solid rgb(211, 215, 207);
	border-radius: 10px;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(63, 67, 68);
	font: 13pt &quot;Ubuntu&quot;;
}

QComboBox::down-arrow{
top: 6px;
left: 6px;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Month">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SignInClicked</sender>
   <signal>clicked()</signal>
   <receiver>booking_dialog_add</receiver>
   <slot>OnAcceptClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnAcceptClicked()</slot>
 </slots>
</ui>
